### DISCUSSION TITLE
How can I quickly add new functionality to the standard control(TimePicker).

### DISCUSSION THREAD

**Discussion by foryura** [AvaloniaUI/Avalonia#6641] [Q&A]
Hello, I want to add the ability to set seconds in the TimePicker, but there is no such thing in the control functionality. Actually, I want to expand the functionality within the framework of my project, but I don’t want to make my control from scratch for this.

**Comment by amwx** [comment]
Unfortunately there's no way to do that without creating your own TimePicker. Adding other time units, selecting duration and things like that have been frequently requested over at WinUI, but there's been no movement on that as far as I'm aware.
